code/code/misc/player_data.cc

```diff
@@ -2277,6 +2277,39 @@ std::vector<sstring> listAccountCharacters(sstring name)
   return list;
 }
 
+int count_set_bit(int n) {
+   int count = 0;
+   while(n != 0) {
+      if((n & 1) == 1) {
+         count++;
+      }
+      n = n >> 1; //right shift 1 bit
```

Comment:
`n >>= 1`

Also, it's very rarely worth commenting what you do (it's redundant with the code, and might get out of sync, confusing everybody).
Instead, comments (if any) should say _why_ you do what you do.

Reply:
Updated this function. Thanks for the review. 

Yeah I use comments as a scratchpad while I working through things and forget to delete them. That was a redundant one.